### DISCUSSION TITLE
Windows Headers Missing via Conda?

### DISCUSSION THREAD

**Discussion by tshauck** [rdkit/rdkit#5972] [Development]
Hi -- I'm trying to build a C++ project that uses rdkit as a dependency that seemingly works on mac, but I'm having trouble getting it to build on windows.

I'm using conda to install rdkit, and on my mac it includes all the header files at `envs/$ENV/include/rdkit`, which I can then include in my project. On my windows machine, it installs minimal headers of `RDGeneral` at `envs\$ENV\Library\include\rdkit`, so then when trying to build my project an error is thrown, e.g. `#include <GraphMol/GraphMol.h>` doesn't exist.

Is this the expected behavior / is there anyway to specify I'd like conda to install all the headers? Some of my windows naivety may be on display :). I'm installing via `conda install -c conda-forge rdkit` in both cases.

**Comment by chryswoods** [comment]
I second this - I have the same problem and can't build a conda package that depends on rdkit because the header files appear to be missing (or I can't find within the conda install). I only see the same minimal headers as @tshauck. Everything works properly on MacOS and Linux. I too just want `GraphMol/GraphMol.h`.

Just to add - thanks for rdkit - it is fantastic :-)

I have access to a windows box and can try and debug this and raise a pull request if that would help?

**Reply by greglandrum** [reply]
This is an issue with the conda-rdkit repo and would best be filed there: https://github.com/conda-forge/rdkit-feedstock

**Comment by chryswoods** [comment]
Thanks - yes, we found the solution. The Windows conda package doesn't include the header files (and other dev files) as they are too large. They have been split out into a `rdkit-dev` package. I can compile if I `conda install -c conda-forge rdkit-dev` (which also, implicitly installs `rdkit`). This is different to the Linux and Mac packages that include the header files by default.

This can be closed. Sorry for sending to the wrong repo.

**Reply by tshauck** [reply]
Thanks for posting your solution.
